templates/tree.html.erb

```diff
@@ -38,8 +38,14 @@
 </style>
 <body>
   <div id="tree-container"></div>
+<% if options[:local_dependencies]%>
+  <% Dir.entries("js").each do |jsfile|%>
+    <%= "<script>#{File.read File.join('js',jsfile)}</script>" if jsfile != '.' && jsfile != '..' %>
+  <%end%>
+<% else%>
```

Comment:
1- Okay. I think a caching policy will satisfies your eagerness towards a clean Ruby gem 😩. When the `-l` argument is used for the first time the gem will download and cache them in `js` directory. What do you think?
2. So you propose that the gem should load the JavaScript source from the `src` tag pointed to a local (cached) directory?